pkgdown/_pkgdown.yml

```diff
@@ -20,29 +20,44 @@ template:
 development:
   mode: auto
 
+articles:
+- title: Customise your site
+  navbar: ~
+  contents:
+  - pkgdown
```

Comment:
This vignette doesn't seem to actually be appearing in the dropdown menu.

<img width="794" alt="Screen Shot 2020-03-17 at 2 10 34 PM" src="https://user-images.githubusercontent.com/599454/76902376-19a76400-6859-11ea-891a-7aa12c0bfcbc.png">

Reply:
Because it has the same name as the package, it's the "get started" entry in the navbar